dc/s2n-quic-dc/events/acceptor.rs

```diff
@@ -17,13 +17,21 @@ struct AcceptorTcpStarted<'a> {
 }
 
 /// Emitted when a TCP acceptor completes a single iteration of the event loop
-#[event("acceptor:tcp:started")]
+#[event("acceptor:tcp:loop_iteration_completed")]
 #[subject(endpoint)]
 struct AcceptorTcpLoopIterationCompleted {
     /// The number of streams that are waiting on initial packets
     #[measure("pending_streams")]
     pending_streams: usize,
 
+    /// The number of slots that are not currently processing a stream
+    #[measure("slots_idle")]
+    slots_idle: usize,
+
+    /// The percentage of slots currently processing streams
+    #[measure("slot_utilization", Percent)]
+    slot_utilization: f32,
```

Comment:
I think this is going to be 0 to 1 today... should this be a Percent then? Maybe we *100 to get 0 to 100?

Reply:
Yeah I was trying to decide if you multiply as part of the event data or the subscriber would know that since it's a `Units::Percent` it needs to decide how to represent it? But thinking about it more it's probably easier to consume if we multiply at the call site.